cni/network/plugin/main.go

```diff
@@ -225,6 +225,10 @@ func main() {
 		panic("network plugin start fatal error")
 	}
 
+	// Check CNI_COMMAD value for logging purposes.
+	cmd := os.Getenv("CNI_COMMAND")
+	log.Errorf("CNI_COMMAND environment variable set to %s", cmd)
```

Comment:
`log.Printf`? This seems more akin to INFO level logging

Reply:
resolved.